src/test/java/ibm/jceplus/junit/base/BaseTestEdDSASignature.java

```diff
@@ -299,7 +300,7 @@ private void runBasicTest(String name, Object param) throws Exception {
 
         Signature sig = Signature.getInstance(name, getProviderName());
         sig.initSign(kp.getPrivate());
-        byte[] testMessage = origMsg;
+        byte[] testMessage = Arrays.copyOf(origMsg, origMsg.length);
```

Comment:
I dont see how the original msg will be modified, any reason why we need to copy it?

Reply:
If we don't copy it, we just create a reference to the original message. And then, if you check line 323 below, you'll see that every time a thread runs this test, it actually changes the original message.